flixel/graphics/FlxAsepriteUtil.hx

```diff
@@ -0,0 +1,442 @@
+	 * empty on every layer) in tags, while it will reduce the size of your json, this method will
+	 * not be able to add those empty frames to an animation.
+	 * 
+	 * @param   sprite     The sprite to add the animations
+	 * @param   data       Can be an `AseAtlas` struct, a JSON string matching the `AseAtlas` or a
+	 *                     string asset path to a json
+	 * @param   tagSuffix  The delimeter on each frame name between the animation name and the
+	 *                     frame number. Be sure to set your filename format to `{outertag}:{frame}`
+	 * @return  This `FlxSprite` instance (nice for chaining stuff together, if you're into that).
+	 * @see flixel.graphics.FlxAsepriteUtil.AseAtlasMeta
+	 * @since 5.4.0
+	 */
+	public static function addAseAtlasTagsByPrefix(sprite:FlxSprite, data:FlxAsepriteJsonAsset, tagSuffix:String = ":")
+	{
+		final aseData = data.getData();
+		for (frameTag in aseData.meta.frameTags)
+			sprite.animation.addByPrefix(frameTag.name, frameTag.name + tagSuffix);
+		
+		return sprite;
+	}
+	
+	/**
+	 * Helper for parsing Aseprite atlas json files. Reads frame data via `loadAseAtlas`,
+	 * then, adds animations for any tags listed, via `addAseAtlasTagsByIndex`.
+	 * 
+	 * Note: It's recommended to disable the "Ignore Empty" option if you have empty frames (that 
+	 * is, empty on every layer) in tags, while it will reduce the size of your json, this method
+	 * will not be able to add those empty frames to an animation.
+	 * 
+	 * @param   sprite     The sprite to load the ase atlas's frames
+	 * @param   graphic    The png file associated with the atlas
+	 * @param   data       Can be an `AseAtlas` struct, a JSON string matching the `AseAtlas` or a
+	 *                     string asset path to a json
+	 * @return  This `FlxSprite` instance (nice for chaining stuff together, if you're into that).
+	 * @see flixel.graphics.FlxAsepriteUtil.AseAtlasMeta
+	 * @since 5.4.0
+	 */
+	public static function loadAseAtlasAndTagsByIndex(sprite:FlxSprite, graphic, data:FlxAsepriteJsonAsset)
+	{
+		final aseData = data.getData();
+		loadAseAtlas(sprite, graphic, aseData);
+		return addAseAtlasTagsByIndex(sprite, aseData);
+	}
+	
+	/**
+	 * Loops through the given ase atlas's tags and adds animations for each, to the given sprite.
+	 * Uses the tag's `to` and `from` fields to determine.
+	 * 
+	 * Note: It's recommended to disable the "Ignore Empty" option if you have empty frames (that 
+	 * is, empty on every layer) in tags, while it will reduce the size of your json, this method
+	 * will not be able to add those empty frames to an animation.
+	 * 
+	 * @param   sprite     The sprite to add the animations
+	 * @param   data       Can be an `AseAtlas` struct, a JSON string matching the `AseAtlas` or a
+	 *                     string asset path to a json
+	 * @return  This `FlxSprite` instance (nice for chaining stuff together, if you're into that).
```

Comment:
can these all just be "The `FlxSprite` (nice for [...]" to avoid potential confusion with the keyword `this`?

Reply:
this was copied from a bunch of FlxSprite methods and the wording is all over, and these methods are meant to be used via static extension  i.e. `using flixel.graphics.FlxAsepriteUtil;` and `sprite.loadAseAtlasAndTags(graphic, data, " ");`

That said, i see no issue in changing "This" to "The"